addons/pos_self_order/__manifest__.py

```diff
@@ -75,7 +75,7 @@
             "pos_self_order/static/src/overrides/components/receipt_header/*",
             "point_of_sale/static/src/app/printer/base_printer.js",
             "point_of_sale/static/src/app/printer/printer_service.js",
-            'web_editor/static/lib/html2canvas.js',
+            'point_of_sale/static/lib/html-to-image.js',
```

Comment:
```suggestion
            'point_of_sale/static/src/app/utils/html-to-image.js',
```

but the file should be in "lib" because we are not the author. If it's in lib folder, we normally avoid touching the file.